.cloudbees/workflows/workflow.yaml

```diff
@@ -37,31 +47,54 @@ jobs:
           service:
             type: ClusterIP
     - name: uninstall a helm chart with dry-run
-      uses: .
+      uses: ./.cloudbees/testing
       with:
         release-name: nginx-example
         namespace: ${{ steps.createns.outputs.name }}
         timeout: 1m
         dry-run: "true"
     - name: uninstall a helm chart
-      uses: .
+      uses: ./.cloudbees/testing
       with:
         release-name: nginx-example
         namespace: ${{ steps.createns.outputs.name }}
         timeout: 1m
```

Comment:
It might be good to verify that the uninstall worked. Something like:
```
 - name: verify remote/OCI chart was removed
    uses: docker://alpine/k8s:1.27.3
    run: |
      set -ux
      EXPECTED='Error from server (NotFound): deployments.apps "nginx" not found'
      ACTUAL="$(kubectl get deployment nginx -n ${{ steps.createns.outputs.name }})"
      if [ "$ACTUAL" != "$EXPECTED" ]; then
        echo "Deployment still exists"
        exit 1
      fi
```

Reply:
skipped this test for now, will work on it after the current version is promoted